controllers/goharbor/harbor/trivy.go

```diff
@@ -75,7 +76,11 @@ func (r *Reconciler) GetTrivyUpdateSecret(ctx context.Context, harbor *goharborv
 
 	token, err := r.GetGithubToken(TrivyGithubCredentialsConfigKey)
 	if err != nil {
-		return nil, serrors.UnrecoverrableError(err, serrors.OperatorReason, "cannot get default github credentials")
+		if _, ok := err.(configstore.ErrItemNotFound); ok {
```

Comment:
Why return nil, nil when it is not found?

Reply:
for trivy, it doesn't have to have a github token

Reply:
I don't think we should check it here. For the case of not setting, we should check the token secret ref in the CR spec.

Reply:
it will find another default token if not set inspec, and if neither is found, it will return ErrItemNotFound

Reply:
Offline discussion with zm, let's keep the way used in the code